src/explore-education-statistics-admin/src/pages/release/datablocks/ReleaseDataBlocksPage.tsx

```diff
@@ -114,116 +114,130 @@ const ReleaseDataBlocksPageInternal = ({
+              id="selectedDataBlock"
+              name="selectedDataBlock"
+              label="Select an existing data block to edit or create a new one"
+              disabled={isLoading}
+              order={[]}
+              value={dataBlockId}
+              optGroups={{
+                'Create data block': [
+                  {
+                    label: 'Create new data block',
+                    value: '',
+                  },
+                ],
+                'Edit existing': dataBlockOptions,
               }}
-            >
-              Delete this data block
-            </Button>
-
-            {deletePlan && (
-              <ModalConfirm
-                title="Delete data block"
-                mounted
-                onConfirm={handleDataBlockDelete}
-                onExit={() => setDeletePlan(undefined)}
-                onCancel={() => setDeletePlan(undefined)}
-              >
-                <p>Are you sure you wish to delete this data block?</p>
-                <ul>
-                  {deletePlan.dependentDataBlocks.map(block => (
-                    <li key={block.name}>
-                      <p>{block.name}</p>
-                      {block.contentSectionHeading && (
-                        <p>
-                          {`It will be removed from the "${block.contentSectionHeading}" content section.`}
-                        </p>
-                      )}
-                      {block.infographicFilesInfo.length > 0 && (
-                        <p>
-                          The following infographic files will also be removed:
-                          <ul>
-                            {block.infographicFilesInfo.map(finfo => (
-                              <li key={finfo.filename}>
-                                <p>{finfo.filename}</p>
-                              </li>
-                            ))}
-                          </ul>
-                        </p>
-                      )}
-                    </li>
-                  ))}
-                </ul>
-              </ModalConfirm>
-            )}
+              onChange={e => {
+                history.push(
+                  generatePath<ReleaseDataBlocksRouteParams>(
+                    releaseDataBlocksRoute.path,
+                    {
+                      publicationId,
+                      releaseId,
+                      dataBlockId: e.target.value ? e.target.value : undefined,
+                    },
+                  ),
+                );
+              }}
+            />
+            <hr />
           </>
         )}
 
-        <ReleaseDataBlocksPageTabs
-          key={selectedDataBlock?.id}
-          releaseId={releaseId}
-          selectedDataBlock={selectedDataBlock}
-          onDataBlockSave={handleDataBlockSave}
-        />
-      </LoadingSpinner>
-    </div>
+        <LoadingSpinner loading={isLoading}>
+          {selectedDataBlock?.name && <h2>{selectedDataBlock.name}</h2>}
```

Comment:
Shouldn't this still say 'Create new data block' if an existing one hasn't been selected?